attachment_helpers.py

```diff
@@ -41,7 +41,7 @@ def get_download_url(video_key):
     return S3_CLIENT.generate_presigned_url(
         "get_object",
         Params={"Bucket": settings.BUCKET_NAME, "Key": video_key},
-        ExpiresIn=60,
+        ExpiresIn=600,
```

Comment:
Will this also affect consent coding? (That'd be great!)

Reply:
yah it will! We should still fix the UI signaling though (for that, this is just kicking the can down the road lol)